.github/workflows/publish-release.yml

```diff
@@ -323,3 +485,40 @@ jobs:
               run: |
                   security delete-keychain $RUNNER_TEMP/app-signing.keychain-db
                   rm -f ~/Library/MobileDevice/Provisioning\ Profiles/build_pp.mobileprovision
+
+    release_web_checksums:
+        name: "Release web checksums"
+        environment: ${{ github.event.inputs.environment || 'Beta' }}
+        needs: create_release_tag
+        env:
+            RELEASE_ID: ${{ needs.create_release_tag.outputs.release_id }}
+            PL_VENDOR_VERSION: ${{ github.event.inputs.vendor_version || '0.0.1' }}
+
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                  node-version-file: ".nvmrc"
+            - name: Install dependencies
+              run: |
+                  npm i -g npm@8.2.0
+                  npm ci
+            - name: Build
+              run: |
+                  npm run pwa:build
+              env:
+                  PL_PWA_URL: ${{ secrets.PL_PWA_URL }}
+                  PL_SERVER_URL: ${{ secrets.PL_SERVER_URL }}
+            - name: Generate checksums
+              run: |
+                  cd packages/pwa/dist
+                  find . -type f ! \( -name "*.txt" -or -name "*.png" -or -name "*.json" -or -name "*.svg" -or -name "*.woff2" -or -name "*.map" \) -exec sha256sum {} > sha256sums-web.txt \;
+            - name: Upload checksums
+              uses: svenstaro/upload-release-action@2.2.1
+              with:
+                  repo_token: ${{ secrets.GITHUB_TOKEN }}
+                  tag: "v${{ env.PL_VENDOR_VERSION }}"
+                  file: packages/pwa/dist/sha256sums-web.txt
+                  asset_name: sha256sums-web.txt
+                  prerelease: true
```

Comment:
Actually, why not release the pwa dist folder as well? For completeness' sake?

Reply:
I actually thought about it, but it'll be built for production/beta, so it won't be "download + usable" by people, which is weird to have, I suppose.

Reply:
Ok, I'll add it.